3.8/aql/fundamentals-data-types.md

```diff
@@ -68,8 +68,20 @@ The following notations are invalid and will throw a syntax error:
 00
 ```
 
-All numeric values are treated as 64-bit double-precision values internally.
-The internal format used is IEEE 754.
+Since ArangoDB 3.8, numeric integer literals can also be expressed as binary
+(base 2) or hexadecimal (base 16) number literals.
+
+- The prefix for binary integer literals is `0b`, e.g. `0b10101110`.
+- The prefix for hexadecimal integer literals i `0x`, e.g. `0xabcdef02`.
+
+Binary and hexadecimal integer literals can only be used for unsigned integers.
+The maximum supported value for binary and hexadecimal numeric literals is
+2<sup>32</sup> - 1, i.e. `0b11111111111111111111111111111111` (binary) or
+`0xffffffff` (hexadecimal).
+
+All numeric values are treated as 64-bit signed integer or 64-bit
+double-precision floating point values internally. The internal floating-point
+format used is IEEE 754.
```

Comment:
@jsteemann Should we add a remark here about V8/JS having a smaller range?

Reply:
@Simran-B : I now added that remark.